aait/web/group-4/starter-project/app/blogs/page.tsx

```diff
@@ -0,0 +1,35 @@
+import BlogCard from "@/components/blogs/BlogCard";
+
+export default function Blogs(){
+    let pages : number[] = [1, 2, 3, 4]
+
```

Comment:
if you want the blog to appear that amount of time you should map through the data not a set of numbers